### PR TITLE
VB-2708 Integration tests for incentive groups

### PR DESCRIPTION
**Pull request by tpmcgowan**
Add integration test for:
* viewing a prison's incentive level groups

Also:
* remove some duplication by putting some re-used methods on top `Page` class
* made naming of some types / test data more consistent

### CODE REVIEW

File: server/routes/testutils/testData.ts [hunk 1]
```diff
@@ -179,11 +179,11 @@ export default class TestData {
     categories = ['A_EXCEPTIONAL', 'A_HIGH', 'A_PROVISIONAL', 'A_STANDARD'],
   }: Partial<CategoryGroup> = {}): CategoryGroup => ({ name, reference, categories })
 
-  static incentiveLevelGroup = ({
+  static incentiveGroup = ({
```
Comment (tpmcgowan) [inline server/routes/testutils/testData.ts:182]: Just making naming consistent - though has caused a fair ripple of changes...

File: server/data/visitSchedulerApiTypes.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@ export type CreateCategoryGroupDto = components['schemas']['CreateCategoryGroupD
 export type PrisonerCategories = CategoryGroup['categories'][number]
 
 // Incentive groups
-export type IncentiveLevelGroup = components['schemas']['SessionIncentiveLevelGroupDto']
+export type IncentiveGroup = components['schemas']['SessionIncentiveLevelGroupDto']
```
Comment (tpmcgowan) [inline server/data/visitSchedulerApiTypes.ts:9]: Making naming of our type consistent with the folder structure, routes, etc

File: integration_tests/pages/page.ts [hunk 3]
```diff
@@ -27,6 +27,10 @@ export default abstract class Page {
 
   manageDetails = (): PageElement => cy.get('[data-qa=manageDetails]')
 
+  successMessage = (): PageElement => cy.get('.moj-banner__message')
```
Comment (tpmcgowan) [inline integration_tests/pages/page.ts:30]: Moving these to top `Page` level so available everywhere

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

**Pull request merged**